repository.datadog.yaml

```
@@ -0,0 +1,10 @@
+# https://datadoghq.atlassian.net/wiki/spaces/DEVX/pages/3176367142/Cleanup+Stale+Branches
+schema-version: v1
+kind: stale-branches
+max_age: 4380h # 6 months
+matches: # keep specific branch patterns from being deleted
+# Protected branches: https://github.com/DataDog/integrations-core/settings/branches
+- master
+- agent-v5
```

nit: I'd even 
```suggestion
- agent-v\d+
```
since we will soon:tm: branch for agent 6 😏